docs/conf.py

```diff
@@ -40,8 +40,8 @@
 master_doc = "index"
 
 # General information about the project.
-project = u"django-fancy-cache"
-copyright = u"2013, Peter Bengtsson"
+project = "django-fancy-cache"
+copyright = "2013, Peter Bengtsson"
```

Comment:
```suggestion
copyright = "2013-2022, Peter Bengtsson"
```
but it's silly because **you**'ve done all the work in the last year! But I guess that's how copyright things work.